docs: threat protection policies: unable to adjust default values (#320)
gravitee-io/release: upgrades/3.x/3.9.4/README.adoc

= Upgrade to 3.9.4

== Breaking changes

From this version, configuration form for JSON Threat Protection Policy and XML Threat Protection Policy changes:

> `null` is not authorized anymore, only `-1` is accepted for a 'no limit' setting.

Reviewer: ```suggestion
`null` is no longer authorized, only `-1` is accepted for a 'no limit' setting.
```

=== Impacts

> - If some fields are set to `null` in your db, and if you want to edit this policy configuration through *API Console*, then the form will replace them by the default value. Please, be careful when updating your policy if you do not want the default values to apply.

Reviewer: ```suggestion
- If some fields are set to `null` in your db and if you want to edit this policy configuration through *API Console*, then the form will replace them by the default value. Please, be careful when updating your policy if you do not want the default values to be applied.
```
